compiler/src/iree/compiler/InputConversion/Common/Passes.td

```diff
@@ -15,6 +15,12 @@ def IREEImportPublic :
   let constructor = "mlir::iree_compiler::createIREEImportPublicPass()";
 }
 
+def ImportMLProgram :
+    Pass<"iree-import-ml-program", "ModuleOp"> {
```

Comment:
(not for this change, but future) From the naming in the file is difficult to identify the naming convention :)